conf/modules.config

```diff
@@ -83,6 +84,40 @@ process {
         ]
     }
 
+    withName: MAGECK_COUNT {
+        publishDir       = [
+            path: { "${params.outdir}/mageck/count/" },
+            saveAs: { filename -> filename.equals('versions.yml') ? null : filename }
+        ]
+        ext.prefix = 'count_table'
+
+    }
+
+    withName: MAGECK_MLE {
+        publishDir       = [
+            path: { "${params.outdir}/mageck/mle/${meta.id}/" },
+            saveAs: { filename -> filename.equals('versions.yml') ? null : filename }
+        ]
+        ext.args   =
+            params.crisprcleanr ? "--norm-method none" : ''
```

Comment:
single quote.

Reply:
This is just a style issue. No need to change.

Reply:
```suggestion
            params.crisprcleanr ? '--norm-method none' : ''
```